openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
+
+    @classmethod
+    def to_json(cls):
```

Comment:
It's a common pattern that `to_string` and `to_json` methods are part of the model itself. It indeed is the one with most knowledge of its state and how to serialise it. Encapsulation trumps [single-responsibility](https://en.wikipedia.org/wiki/Single_responsibility_principle) in that case, as a matter of [cohesion](https://en.wikipedia.org/wiki/Cohesion_(computer_science)).